src/aks-preview/azext_aks_preview/_consts.py

```diff
@@ -12,6 +12,12 @@
 CONST_SPOT_EVICTION_POLICY_DELETE = "Delete"
 CONST_SPOT_EVICTION_POLICY_DEALLOCATE = "Deallocate"
 
+# consts for upgrade channel
+CONST_RAPID_UPGRADE_CHANNEL = "rapid"
+CONST_STABLE_UPGRADE_CHANNEL = "stable"
+CONST_PATCH_UPGRADE_CHANNEL = "patch"
+CONST_NONE_UPGRADE_CHANNEL = "none"
```

Comment:
Are these values defined in the swagger and have been defined as Enum in the SDK ? If yes, we may be able to leverage it instead of defining separately.

Reply:
they are defined in the swagger, however, consts file does depend on the versioned SDK, not sure if it's the reason that we are defining these consts here (don't want to introduce dependency?). Probably I am going to break the pattern and keep it consistent for now. :)